e2e/stack_test.go

```diff
@@ -76,6 +80,22 @@ func TestE2E(t *testing.T) {
 
 	const targetHeight = 5
 
+	client, err := cosclient.New(monomerCometURL.String(), monomerCometURL.String())
+	require.NoError(t, err, "failed to create Comet client")
```

Comment:
nit: from what I have seen, error strings state the intended action. Only logs include `Failed to`
```suggestion
	require.NoError(t, err, "create Comet client")
```

Reply:
e6b9ef6